### ISSUE TITLE
Adjusting keyboard height should also increase size of Symbols

### ISSUE THREAD

**Issue by HyknuF** [florisboard/florisboard#32]
Thanks again for your nice work.  It looks, that adjusting keyboard height only affects the surrounding button size. I'll suggest that it should increase the font size of the symbols as well. 

Thank you!

Edit:  Seeing another screenshots, maybe it's  bug on my device?

**Environment information**

FlorisBoard Version: 0.2.2
Install Source: izzy f-droid
Device: OnePlus 6
Android version, ROM: OmniRom android 10

**Steps to reproduce**

Changing Keyboard height in Settings doesn't affect the font size of symbols.

Screenshots  are showing  extra short and extra tall

![Screenshot_Noteless_20201031-112350](https://user-images.githubusercontent.com/53949193/97776818-b2ca8700-1b6b-11eb-8ede-ab17c46ae6e6.png)
![Screenshot_Noteless_20201031-112311](https://user-images.githubusercontent.com/53949193/97776819-b3fbb400-1b6b-11eb-9031-891aaae6471a.png)

**Comment by patrickgold** [comment]
You are indeed correct, the font size is not affected by the keyboard height. What is affected though is the size of the icons of the delete key, shift, enter, etc.

> Edit: Seeing another screenshots, maybe it's bug on my device?

Other screenshots of FlorisBoard have been taken on different devices, where the system font size may be at a different value and thus it looks different, so it isn't a bug on your device.

I will look into it and make the font size adapt to the key size. Thanks for your suggestion!

**Comment by HyknuF** [comment]
Glad to hear that, thanks! I am really looking forward to v0.3.0 :-D 👍

**Comment by patrickgold** [comment]
Glad to hear that you are looking forward to v0.3.0! 😄 Above commit adds auto font sizing, which should increase the font size accordingly to the keyboard height preference. Will be included in the next release (v0.2.3). Note that I will let this issue open until you have checked and verified that the auto font-sizing actually works on your device too, so I can assure that my implementation actually works.

**Comment by HyknuF** [comment]
Glad to hear that! I will give feedback for sure. Are you going to push the 0.2.3. apk via izzy/fdroid  or github-repo?

**Comment by patrickgold** [comment]
The releases of versions are synced between github and izzy/fdroid. It only takes a little bit of time until a new version arrives in izzy/fdroid, while on github the version is immediately available after I release it.

**Comment by HyknuF** [comment]
Just curious: Do you already know when you will upload v0.2.3?   :-)

**Comment by patrickgold** [comment]
Sorry for responding so late, I'm currently a bit spare on free time. I'll probably release v0.2.3 in the next few days (maybe even today if my new core implementation finally works like it should...). Stay tuned! :-)

**Comment by patrickgold** [comment]
I've just released v0.2.3. Should arrive at your device soon. Could you please let me know if the font size adjusting works on your device when the new version arrives at your device? Thanks in advance and have a nice evening!

**Comment by HyknuF** [comment]
Thanks for updating! I've just installed v0.2.3; on my device it works as desired. See screenshot below, smallest and tallest setting. For comparison, I added a screenshot of Openboard in default size as well.
![Screenshot_Noteless_20201112-063907](https://user-images.githubusercontent.com/53949193/98901148-8ab81d80-24b3-11eb-9b36-a3036e84a676.png)
![Screenshot_Noteless_20201112-063946](https://user-images.githubusercontent.com/53949193/98901150-8be94a80-24b3-11eb-820e-7b8d7728bfa3.png)
![Screenshot_Noteless_20201112-064137](https://user-images.githubusercontent.com/53949193/98901154-8c81e100-24b3-11eb-8276-efdc08840f80.png)
